recipes/namedtype/all/conanfile.py

```diff
@@ -18,9 +18,13 @@ def _source_subfolder(self):
     def source(self):
         tools.get(**self.conan_data["sources"][self.version])
         url = self.conan_data["sources"][self.version]["url"]
-        extracted_dir = "NamedType-" + os.path.splitext(os.path.basename(url))[0]
+        if self.version == "20190324":
+            # non-release version
+            extracted_dir = "NamedType-" + os.path.splitext(os.path.basename(url))[0]
+        else:
+            extracted_dir = self.name + "-" + self.version
         os.rename(extracted_dir, self._source_subfolder)
 
     def package(self):
-        self.copy("*.hpp", dst="include", src=self._source_subfolder)
+        self.copy("*.hpp", dst="include", src=self._source_subfolder, keep_path=False)
```

Comment:
I would keep `NamedType` folder in 1.1.0, add it for the older version, and add this path int package_info.

Reply:
Seems that upstream example expects to find headers in `NamedType` subdirectory. I updated the recipe to install to this directory regardless of version.